### ISSUE TITLE
Choked Cymbals Notation

### ISSUE THREAD

**Issue by Dima12357** [musescore/MuseScore#21688]
### Your idea

I suggest implementing the correct automatic placement of the "Choke (Weinberg)" symbol and the corresponding sound that should be obtained as a result of adding this instruction.
![Снимок экрана (112)](https://github.com/musescore/MuseScore/assets/134041121/c0cf38f0-0e16-4406-b2be-bcb2a59f3f7e)

### Problem to be solved

Currently, Musescore 4 does not have the correct automatic placement of this symbol. The most important thing is that adding this symbol currently does not change the sound of the cymbals in any way.

**Comment by zacjansheski** [comment]
Related to https://github.com/musescore/MuseScore/issues/19753

What is the issue with placement of the breath mark specifically?  The vertical position?

**Comment by Dima12357** [comment]
> Относится к #19753
> 
> В чем конкретно проблема с размещением отметки дыхания? Вертикальное положение?

![Снимок экрана (114)](https://github.com/musescore/MuseScore/assets/134041121/bd95d076-cf02-4cbd-b1ea-f975993f2a74)
Issue #19753 , which you mentioned, contains a proposal to create a damping capability for individual instruments. Damping and Choking techniques are completely different techniques! Damping occurs only after holding the full duration of the note. And when choking, damping occurs at the expense of the preceding note (while the full duration is not maintained) - this is the whole essence of the choking technique and the difference with damping. 
And as for damping, I created a separate issue on this topic: #21685 . Currently, in Musescore, percussion and harp sounds automatically continue to vibrate after the note/chord duration has been fully sustained. This shouldn't happen. And for those who think that this requires some additional instruction, I recommend reading Elaine Gould's guide to musical notation. Instructions for damping are not required. On the contrary, the only permission for natural sustained resonance should be the instruction "laissez vibrer".

**Comment by zacjansheski** [comment]
Please keep the issue focused on one thing for developers.

Let's keep this strictly an engraving conversation from now on, thank you!

**Comment by Dima12357** [comment]
> Пожалуйста, сосредоточьте проблему на чем-то одном для разработчиков.
> 
> Давайте впредь вести этот разговор строго гравюрно, спасибо!

I also found it important to point out that in addition to the correct engraving, it is necessary to implement the sound itself in the style of Choked Cymbal. Are you suggesting that I make a separate issue about the sound? Isn't it possible to combine the two into one?

**Comment by zacjansheski** [comment]
We generally seperate playback and engraving work on the dev side of things, so it is best to separate those issues as well.

It would be better to add a note to a related issue like https://github.com/musescore/MuseScore/issues/19753 and now https://github.com/musescore/MuseScore/issues/21685 so related work can be done at the same time
